homeassistant/components/device_tracker/icloud.py

```diff
@@ -78,6 +90,10 @@ def update_icloud(now):
         except PyiCloudNoDevicesException:
             _LOGGER.info('No iCloud Devices found!')
 
+    hass.services.register('device_tracker',
```

Comment:
the service isn't registered per device, but per account. you can't update just one device, you can only update all devices linked to one account